apps/berlinc/berlin-clock.js

```diff
@@ -1,3 +1,6 @@
+// timeout used to update every minute
+let drawTimeout;
```

Comment:
Out of interest, why did you put this here and then change to use `global.drawTimeout`? Every other clock I've seen just put it inside the curly braces and it's fine?

Reply:
Hm, the thought was that the remove method and timeouts are called from global scope and would then not be able to find a non-global var. Did not explicitly test that, should it work?

Reply:
It should - but IMO 'if it's not broken, don't fix it'. It'd be good if you could just keep the pattern the same as other clocks.

In JavaScript a function has access to all variables in the scope it's defined in, so because `remove` is defined inside the curly braces too, it's fine.

Reply:
You're right, seems to work just fine inside the scope.